_episodes/02-project-setup.md

```diff
@@ -248,10 +248,20 @@ For reading, check out the [curriculum](https://mozilla.github.io/open-leadershi
 Clean up and organize content: where to different pieces of code go -->
 
 > ## Practice working on projects
-> FIXME: provide a example project folder, spend time sorting,
-> or allow people some time to work on their own projects and generating questions.
+> To practice organising a project, download the [Gapminder example project][example-project] and spend a few minutes
+> organising it: create directories, move and rename files, and even edit the code if needed. Be sure to have a glance
+> at the content of each file. To download it, either use [this link][example-project-zip] to download a ZIP file or
+> clone the repository.
+> 
+> > ## Solution
+> > A possible way to organise the project is in the project's `tidy` branch. [Click here][tidy-branch] to open it on
+> > GitHub. Discuss: what is different from the version you downloaded?
```

Comment:
Click here is not good link text, also can we include a few key points about the solution here?

Reply:
Thank you for the review, Sarah. I've applied the requested changes.